packages/cli/src/config/validation.ts

```diff
@@ -151,3 +151,13 @@ export function validateSelector(name: string, ctx: RefinementCtx) {
     });
   }
 }
+
+/** Returns null if the type does not look like a static array, otherwise element and length data */
+export function parseStaticArray(abiType: string) {
+  const matches = abiType.match(/^(\w+)\[(\d+)\]$/);
+  if (matches === null) return null;
```

Comment:
slight preference for `== null` here to catch `undefined` (I know the docs say this should return `null` but just wanna be safe)

Reply:
changed it to just `!matches`, might as well catch any falsey value then